apps/account-api/src/controllers/health.controller.ts

```diff
@@ -23,7 +19,7 @@ export class HealthController {
   }
 
   // Live endpoint
-  // eslint-disable-next-line class-methods-use-this
```

Comment:
Some of these rules are disabled in so many places, it seemed pointless to turn the rule on.